rai/lib/utility.cpp

```diff
@@ -1,13 +1,21 @@
 #include <iostream>
 #include <rai/lib/utility.hpp>
 
+#if ACTIVE_NETWORK == rai_test_network
+/*
+ * If we are running under the test harness, create this as a
+ * weak symbol so the one we provide as part of gtest can
+ * override it.
+ */
+#  pragma weak release_assert_internal
```

Comment:
Does msvc support this?

Reply:
I don't think MSVC supports it, but my understanding is that it will ignore the unknown pragma and during linking it will pick one of the symbols.  If it picks the less-than-optimal one on MSVC then there will slightly less optimal tests, but overall there will be no change.